docs/contributing/nova.md

```diff
@@ -0,0 +1,125 @@
+# Nova adoption
+
+## Prerequisites
+
+* Previous Adoption steps completed. Notably, the service databases
+  must already be imported into the podified MariaDB.
+
+## Variables
+
+(There are no shell variables necessary currently.)
+
+## Procedure - Nova adoption
+
+* Patch OpenStackControlPlane to deploy Nova:
+
+  ```
+  oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
+  spec:
+    nova:
+      enabled: true
```

Comment:
I think you can go with the default greenfield yaml [snippet](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_galera_network_isolation.yaml#L136C1-L161C25) for nova,which is
```
  nova:
    enabled: true
    apiOverride:
      route: {}
    template:
      apiServiceTemplate:
        override:
          service:
            internal:
              metadata:
                annotations:
                  metallb.universe.tf/address-pool: internalapi
                  metallb.universe.tf/allow-shared-ip: internalapi
                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
              spec:
                type: LoadBalancer
      metadataServiceTemplate:
        override:
          service:
            metadata:
              annotations:
                metallb.universe.tf/address-pool: internalapi
                metallb.universe.tf/allow-shared-ip: internalapi
                metallb.universe.tf/loadBalancerIPs: 172.17.0.80
            spec:
              type: LoadBalancer
      secret: osp-secret
```

The rest will be defaulted correctly if:
* the adopted db schemas are called nova_api, nova_cell0 and nova_cell1 respectively. 
* the nova_api and nova_cell0 schema is copied into to the MariaDB/openstack  DB server
* the nova_cell1 is copied into MariaDB/openstack-cell1 DB servers.

Reply:
The default replicas will be correct for a source system where:
* vncproxy was enabled
* nova-metadata was deployed on the top level and not per cell separately.

if the source system is configured differently then we have two choices:
* a) we accept that the configuration of the system will change during adoption.  (E.g. if the vnc was not enabled it will be enabled now)
* b) we inspect the source system, check how these optional services are configured, and create the dest system with a matching service configuration.

Reply:
i belive we only supported toplevel deployment in tripleo
and vnc should be enabled by default

so i think its highly likely that the default sample will be correct in that regard

Reply:
So I don't need all the things below? we have them in isol-net CR example of opensntack operator